lib/sles4sap/sdaf_deployment_library.pm

```diff
@@ -414,6 +415,9 @@ sub set_common_sdaf_os_env {
     $args{sdaf_tfstate_storage_account} //= get_required_var('SDAF_TFSTATE_STORAGE_ACCOUNT');
     $args{sdaf_key_vault} //= get_required_var('SDAF_KEY_VAULT');
 
+    # This is used later filling up tfvars files.
```

Comment:
Only a temporary solution, tfvars templates will be reworked once the whole base deployment works